README.md

```diff
@@ -64,65 +79,86 @@ apply the proper context:
 $ chcon -R system_u:object_r:admin_home_t:s0 docker-elk/
 ```
 
-### Docker for Windows
+### Docker for Desktop
+
+#### Windows
+
+Ensure the [Shared Drives][win-shareddrives] feature is enabled for the `C:` drive.
 
-If you're using Docker for Windows, ensure the "Shared Drives" feature is enabled for the `C:` drive (Docker for Windows > Settings > Shared Drives). See [Configuring Docker for Windows Shared Drives](https://blogs.msdn.microsoft.com/stevelasker/2016/06/14/configuring-docker-for-windows-volumes/) (MSDN Blog).
```

Comment:
Is this outdated?

Reply:
I just rephrased it and changed the link to point to the official documentation, which didn't exist back then.

Reply:
Oh sorry, missed the link, LGTM